core/src/main/java/org/wso2/carbon/kernel/securevault/SecureVaultUtils.java

```diff
@@ -162,7 +162,7 @@ private static String readFromSystem(String alias) {
      *
      * @param value a string that contains placeholders which is needed to get substitute with proper values
      * @return updated String
-     * @throws SecureVaultException
+     * @throws SecureVaultException if the value for placeholder is not specified.
```

Comment:
There are pending javadoc in secure vault feature. This is being handled in a separate PR, therefore shall we revert changes in this file.